src/_main.js

```diff
@@ -238,6 +312,14 @@
 
             if (!this.visible) { return; }
             var _this = this;
+			
+			// set focus back to element that had it before the modal was opened
+			this.focusedElementBeforeModal.focus();
+			
+			if (this.options.ariaPageContent) {
+				jQuery(this.options.ariaPageContent).attr("aria-hidden", "false");
```

Comment:
Line is too long.<br>Strings must use singlequote.<br>Too many errors. (70% scanned).